src/components/ScreeningsSection.vue

```diff
@@ -25,53 +25,75 @@
   </div>
 </template>
 
-<script>
+<script lang="ts">
+import Vue from "vue";
 import BaseSelect from "./base/BaseSelect.vue";
 import ScreeningCard from "./ScreeningCard.vue";
 import { mapGetters } from "vuex";
 import dayTabsMixin from "@/mixins/dayTabs.js";
 import screeningsList from "@/mixins/screeningsList.js";
 
-export default {
+interface Movie {
+id: number;
+title: string;
+poster_url: string; 
+length: number; 
+release_date: string;
+description: string;
+genre: Genre;
+}
+
+interface Genre {
+  id: number; 
+  name: string;
+}
+
+interface Screening {
+id: number;
+datetime: string;
+movie: number;
+hall: number;
+}
+
+interface DataType {
+  query: string;
+  selectedOption: string
+}
+
```

Comment:
What do you think about moving all those `interfaces` into a separate file and then just importing them into this component? :) 
For instance, create a new file called ` interfaces.ts` in the `api` folder and move all declared here interfaces there, then import data types that you need in this component like this:
```
import { Movie, Genre, Screening } from 'filePath';
```